rgo/component.py

```diff
@@ -98,32 +98,36 @@ def _make_srpm(tmpdir, distgit, source_code_git, pkg_name, version_from, patches
         spec_name = distgit.spec_path or "{!s}.spec".format(pkg_name)
 
         # extract spec file from specified branch and save it in temporary location
-        spec = os.path.join(workdir, "original.spec")
-        with open(spec, "w") as f_spec:
+        original_spec_path = os.path.join(workdir, "original.spec")
+        with open(original_spec_path, "w") as f_spec:
             subprocess.run(["git", "cat-file", "-p", "{}:{}".format(distgit.ref, spec_name)],
                            cwd=distgit.cwd, check=True, stdout=f_spec)
 
-        rpmspec = rpm.spec(spec)
+        original_rpmspec = rpm.spec(original_spec_path)
 
-        spec_name = rpmspec.sourceHeader["Name"]
+        spec_name = original_rpmspec.sourceHeader["Name"]
         if isinstance(spec_name, bytes):
             spec_name = spec_name.decode("utf-8")
 
-        spec_version = rpmspec.sourceHeader["Version"]
+        spec_version = original_rpmspec.sourceHeader["Version"]
         if isinstance(spec_version, bytes):
             spec_version = spec_version.decode("utf-8")
 
-        _spec_path = os.path.join(workdir, "{!s}.spec".format(spec_name))
+        _final_spec_path = os.path.join(workdir, "{!s}.spec".format(spec_name))
```

Comment:
Since you're renaming... what's the point of having some variables prefixed with an underscore?